dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/container/config/VirtualHost.java

```diff
@@ -46,8 +48,8 @@ public String toString() {
         if (this.hostAliases != null)
             for (String hostAlias : hostAliases)
                 buf.append("hostAlias=\"" + hostAlias + "\" ");
-        if (this.allowFromEndpoint != null) {
-            buf.append("allowFromEndpoint=\"" + this.allowFromEndpoint + "\" ");
+        if (this.allowFromEndpointRef != null) {
+            buf.append("allowFromEndpointRef=\"" + this.allowFromEndpointRef + "\" ");
         }
         buf.append("}");
 
```

Comment:
Runtime updates making the virtualHost configuration support the allowFromEndpointRef attribute rather than allowFromEndpoint subelement look good and agree with Liberty HTTP Transport configuration descriptors.